src/main/kotlin/com/terraformation/backend/email/EmailService.kt

```diff
@@ -109,6 +109,30 @@ class EmailService(
     }
   }
 
+  /**
+   * Sends an email notification to all users.
+   *
+   * @param [model] Model object containing values that can be referenced by the template.
+   * @param [requireOptIn] If false, send the notification even if the user has not opted into email
+   *   notifications. The default is to obey the user's notification preference, which is the
+   *   correct thing to do in the majority of cases.
+   */
+  fun sendAllUsersNotification(
+      model: EmailTemplateModel,
+      requireOptIn: Boolean = true,
+  ) {
+    val allUsers = userStore.fetchUsers(requireOptIn)
+    allUsers.forEach { user ->
+      if (requireOptIn && !user.emailNotificationsEnabled) {
+        log.info(
+            "Skipping email notification for user ${user.userId} because they didn't enable it")
+      } else {
+        val locale = user.locale ?: Locale.ENGLISH
+        locale.use { send(model, listOf(user.email)) }
```

Comment:
@sgrimm locally, my first user email is "system" so this is failing with error "invalid email", and looks like it doesn't continue with the others users. How can I make that continue ?